iosHackthonApp/Modules/Features/Login/Sources/Login/LoginView.swift

```diff
@@ -7,28 +7,36 @@
 //
 
 import SwiftUI
+import shared
+import Components
 import Strings
 import Theming
-import Components
 import TabBar
 import SignUp
 
 public struct LoginView: View {
-	@State var email: String = ""
-	@State var password: String = ""
-	@State var loginSucessful: Bool = true
-	let coloredNavAppearance = UINavigationBarAppearance()
-
-	public init() {
+	@State private var email: String = ""
+	@State private var password: String = ""
+	@State private var showingAlert = false
+	@State private var activeAlert: ActiveAlert = .collection
+	@State var isLoading: Bool = false
+	private let coloredNavAppearance = UINavigationBarAppearance()
+	private let firebase: FirebaseAuthenticationStore
+	@ObservedObject private var authorizationStore: AuthorizationStore
```

Comment:
It feels like a lot of this stuff should go in a view model? maybe it's not possible, but it might make this view a bit smaller. whatever we can do to try and make SwiftUI views smaller and more reusable the better. I find them quite hard to review in PRs because of their size

Reply:
I agree. We could use `MealListViewModel` as an example :-)

Reply:
yup will do..😅

Reply:
Moved it all to `LoginViewModel`